public/javascripts/SVValidate/src/modal/ModalComment.js

```diff
@@ -11,6 +11,11 @@ function ModalComment (modalUI) {
         disableClickOk: true
     };
 
+    // Initializing feedback popover 
+    $(function () {
```

Comment:
Another example of wrapping something in a function that I don't think you need to do

Reply:
but lmk if I'm wrong about that!